View/AggregatedStageDamage/CalculatedStageDamageControl.xaml.cs

```diff
@@ -19,7 +19,57 @@ private void calculate_btn_Click(object sender, RoutedEventArgs e)
         {
             if (DataContext is CalculatedStageDamageVM vm)
             {
-                vm.CalculateCurves();
+                vm.ComputeCurves();
+            }
+        }
+
+        /// <summary>
+        /// This is to stretch the listview columns to fill the space as the user stretches the window.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void ListView_SizeChanged(object sender, SizeChangedEventArgs e)
+        {
+            ListView listView = sender as ListView;
+            GridView gView = listView.View as GridView;
+            
+            // take into account vertical scrollbar
+            var workingWidth = listView.ActualWidth - SystemParameters.VerticalScrollBarWidth - 5; 
+            var col1 = 0.33;
+            var col2 = 0.33;
+            var col3 = 0.33;
+
+            if (workingWidth > 0)
+            {
+                gView.Columns[0].Width = workingWidth * col1;
+                gView.Columns[1].Width = workingWidth * col2;
+                gView.Columns[2].Width = workingWidth * col3;
+            }
+        }
+
+        /// <summary>
+        /// This is to stretch the listview columns to fill the space as the user stretches the window.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void ListView_SizeChanged_1(object sender, SizeChangedEventArgs e)
```

Comment:
Is this the same code duplicated with a _1?

Reply:
yes, these methods are auto generated events from the .xaml class. I mothed the central logic into its own method and call it from the two events. I changed the names to be more clear about which goes to which list view.